src/commcare_cloud/commands/terraform/constants.py

```diff
@@ -6,10 +6,10 @@
 COMMCAREHQ_XML_POST_URLS_REGEX = r"""
 ^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$|^/a/([\w\.:-]+)/apps/([\w-]+)/multimedia/uploaded/app_logo/([\w\-]+)/$|^/a/([\w\.:-]+)/apps/([\w-]+)/multimedia/uploaded/image/$
 ^/a/([\w\.:-]+)/apps/edit_form_attr/([\w-]+)/([\w-]+)/([\w-]+)/$|^/a/([\w\.:-]+)/apps/edit_form_attr_api/([\w-]+)/([\w-]+)/([\w-]+)/$|^/a/([\w\.:-]+)/apps/patch_xform/([\w-]+)/([\w-]+)/$
-^/a/([\w\.:-]+)/cloudcare/api/readable_questions/$|^/a/([\w\.:-]+)/cloudcare/apps/report_formplayer_error|^/a/([\w\.:-]+)/data_dictionary/import$|^/a/([\w\.:-]+)/fixtures/edit_lookup_tables/upload/$
-^/a/([\w\.:-]+)/fixtures/fixapi/|^/a/([\w\.:-]+)/importer/excel/bulk_upload_api/$|^/a/([\w\.:-]+)/importer/excel/config/$|^/a/([\w\.:-]+)/receiver/$|^/a/([\w\.:-]+)/receiver/([\w-]+)/$
-^/a/([\w\.:-]+)/receiver/secure/$|^/a/([\w\.:-]+)/receiver/secure/([\w-]+)/$|^/a/([\w\.:-]+)/receiver/submission/?$|^/a/([\w\.:-]+)/settings/users/commcare/upload/$|^/formplayer/new-form$
-^/formplayer/validate_form$|^/gvi/api/sms/$|^/jserror/$|^/telerivet/in/?$
+^/a/([\w\.:-]+)/cloudcare/api/readable_questions/$|^/a/([\w\.:-]+)/cloudcare/apps/report_formplayer_error|^/a/([\w\.:-]+)/data/export/custom/download_data_files/$
+^/a/([\w\.:-]+)/data_dictionary/import$|^/a/([\w\.:-]+)/fixtures/edit_lookup_tables/upload/$|^/a/([\w\.:-]+)/fixtures/fixapi/|^/a/([\w\.:-]+)/importer/excel/bulk_upload_api/$
+^/a/([\w\.:-]+)/importer/excel/config/$|^/a/([\w\.:-]+)/receiver/$|^/a/([\w\.:-]+)/receiver/([\w-]+)/$|^/a/([\w\.:-]+)/receiver/secure/$|^/a/([\w\.:-]+)/receiver/secure/([\w-]+)/$
+^/a/([\w\.:-]+)/receiver/submission/?$|^/a/([\w\.:-]+)/settings/users/commcare/upload/$|^/formplayer/new-form$|^/formplayer/validate_form$|^/gvi/api/sms/$|^/jserror/$|^/telerivet/in/?$
```

Comment:
Could this be wrapped to one sub-pattern per line rather than using `|` to put multiple per line? This seems much easier to read.
```suggestion
^/a/([\w\.:-]+)/cloudcare/api/readable_questions/$
^/a/([\w\.:-]+)/cloudcare/apps/report_formplayer_error
^/a/([\w\.:-]+)/data/export/custom/download_data_files/$
^/a/([\w\.:-]+)/data_dictionary/import$
^/a/([\w\.:-]+)/fixtures/edit_lookup_tables/upload/$
^/a/([\w\.:-]+)/fixtures/fixapi/
^/a/([\w\.:-]+)/importer/excel/bulk_upload_api/$
^/a/([\w\.:-]+)/importer/excel/config/$
^/a/([\w\.:-]+)/receiver/$
^/a/([\w\.:-]+)/receiver/([\w-]+)/$
^/a/([\w\.:-]+)/receiver/secure/$
^/a/([\w\.:-]+)/receiver/secure/([\w-]+)/$
^/a/([\w\.:-]+)/receiver/submission/?$
^/a/([\w\.:-]+)/settings/users/commcare/upload/$
^/formplayer/new-form$
^/formplayer/validate_form$
^/gvi/api/sms/$
^/jserror/$
^/telerivet/in/?$
```
I think `^/a/([\w\.:-]+)/receiver/secure/$` can be removed since it is matched by the expression immediately before it.

Reply:
Yeah at some point in the process we need to "compact" these into as few regexes as possible while keeping each regex under a limit to meet AWS WAF constraints, but I agree it would be nicer to do that "compaction" of the regexes after this point rather than in `./manage.py list_waf_allow_patterns --compact`. Will try and make that change soon